pkg/user/tx_client_test.go

```diff
@@ -45,6 +47,9 @@ func (suite *TxClientTestSuite) SetupSuite() {
 	suite.Require().NoError(err)
 	suite.txClient, err = user.SetupTxClient(suite.ctx.GoContext(), suite.ctx.Keyring, suite.ctx.GRPCClient, suite.encCfg, user.WithGasMultiplier(1.2))
 	suite.Require().NoError(err)
+	// FIXME: Temporary way of querying the raw log.
+	// TxStatus will natively support this in the future.
+	suite.serviceClient = sdktx.NewServiceClient(suite.ctx.GRPCClient)
```

Comment:
Well we will still need this to get the gas wanted and gas used

Reply:
true i think i automatically copied this. will remove!